app/frontend/src/components/FormList.vue

```diff
@@ -18,7 +18,11 @@
                         <div>VIEW</div>
                       </v-btn>
                     </router-link>
-                    <router-link class="pr-2" :to="{ name: form.linkAdminName }">
+                    <router-link
+                      v-if="hasResourceRoles(form.resource, form.roles)"
```

Comment:
FYI: in the auth store i'm seeing:
`
if (!roles.length) return true; // No roles to check against
`
as they are logged in to admin area they have a role so that's ok, correct?

Reply:
nevermind. i realise this is a public page now

Reply:
That's by design - the roles parameter represents the set of roles that need to be checked against. If you're checking nothing, then it's allowed as you're not enforcing anything.